build.gradle.kts

```diff
@@ -69,6 +69,7 @@ dependencies {
 
 
     implementation(libs.freemarker)
+    implementation("io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations:1.26.0")
 
```

Comment:
why is this necessary?  will it be packaged in the plugin zip? if yes then its not acceptable because it may clash with jetbrains dependencies.
in nay case , dependencies should be added to settings.gradle.kts like all our other dependencies.